core/src/com/treode/notify/Notification.scala

```diff
@@ -61,11 +62,13 @@ object Notification {
     def messages = List.empty
   }
 
-  /** Easily create NoErrors with the given `result`. */
-  def empty [A] (result: A): Notification [A] = NoErrors (result)
-
-  /** Easily create NoErrors with Unit. */
+  /** Easily create a NoErrors with Unit. */
   def empty: Notification [Unit] = NoErrors (())
 
-  def newBuilder: Builder [Unit] = new Builder ()
+  /** Easily build Errors with varargs. */
+  def buildErrors (messages: Message*) : Notification [Unit] =
```

Comment:
This is cool. The Scala idiom would be to call this method `apply`. Define it before empty.